### DISCUSSION TITLE
Asking about the difference between the "uniform profile" option and the default (Fickian diffusion) in DFN.

### DISCUSSION THREAD

**Discussion by CaritasPostech** [pybamm-team/PyBaMM#2741] [Q&A] [answered]
First, I just want to say thanks to all the members who contributed to Pybamm community.

I'm curious about the difference between the "uniform profile" option and the default Fickian diffusion for particles in pybamm.lithium_ion.DFN. I searched for information about the "uniform profile" in the code, but couldn't find much, so I'm asking this question.

**Comment by rtimms** [comment, marked as answer]
Uniform profile assumes the concentration is uniform within the particle, whereas Fickian diffusion solves a PDE for the concentration within each particle as a function of r and t. 

You can also do quadratic and quartic profiles. See [Subramanian, 2005](https://iopscience.iop.org/article/10.1149/1.2032427) for details.

**Reply by CaritasPostech** [reply]
Thank you for your reply!
